libmamba/include/mamba/specs/regex_spec.hpp

```diff
@@ -75,4 +75,13 @@ struct fmt::formatter<mamba::specs::RegexSpec>
     format(const ::mamba::specs::RegexSpec& spec, format_context& ctx) const -> decltype(ctx.out());
 };
 
+template <>
+struct std::hash<mamba::specs::RegexSpec>
+{
+    auto operator()(const mamba::specs::RegexSpec& spec) const -> std::size_t
+    {
+        return std::hash<std::string>{}(spec.str());
+    }
+};
+
```

Comment:
I didn't check thoroughly all what have been changed with #3363 compared to before, so was this something missing from #3363?

Reply:
This was missing from #3363, yes.